spa/src/components/pageEditor/elementEditors/amount/AmountItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { ButtonBase } from '@material-ui/core';
+import PropTypes, { InferProps } from 'prop-types';
+import { MouseEvent } from 'react';
+import { NumberButton, RemoveIcon, Root } from './AmountItem.styled';
+
+const AmountItemPropTypes = {
+  amount: PropTypes.number.isRequired,
+  isDefault: PropTypes.bool,
+  onRemove: PropTypes.func.isRequired,
+  removable: PropTypes.bool,
+  onSetDefault: PropTypes.func.isRequired
+};
+
+export interface AmountItemProps extends InferProps<typeof AmountItemPropTypes> {
+  onRemove: () => void;
+  onSetDefault: (event: MouseEvent) => void;
+}
+
+export function AmountItem({ amount, isDefault, onRemove, onSetDefault, removable }: AmountItemProps) {
+  return (
+    <Root $isDefault={!!isDefault}>
```

Comment:
Maybe add a default?

```suggestion
export function AmountItem({ amount, isDefault = false, onRemove, onSetDefault, removable }: AmountItemProps) {
  return (
    <Root $isDefault={isDefault}>
```

Reply:
Hmm, when I try that I get this complaint: `Type 'boolean | null' is not assignable to type 'boolean'.` I can change it to `isDefault!` but that seems like it defeats the purpose of trying to avoid a type coercion.